recipes/sdf/all/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def requirements(self):
+        self.requires("stb/20200203")
```

Comment:
Is this external requirement strictly needed? I'd rather just test the library `sdf` here isolated from other packages if possible

Reply:
I just use it to load the png file.

Reply:
I understand. However, this introduces a new dependency in test_package that needs to be taken into account if we want to build or check the ConanCenter graph running the tests_packages of all recipes. Of course this is speculation right now, but I think it is important to keep in mind. I am fine merging this, just wanted to ask if there was an easy alternative fo the example, as it doesn't have to be an exhaustive one, just test some classes of the library a little bit without loading an image.

Reply:
SDF has only 3 functions (https://github.com/memononen/SDF/blob/master/src/sdf.h#L46) that require image data. So it need a 3rdparty lib for extracting raw data and some info like width, height and bpp.

Reply:
ok, let's go for it then